src/core/qgsgdalutils.h

```diff
@@ -66,7 +66,7 @@ class CORE_EXPORT QgsGdalUtils
      * \returns TRUE on success
      * \since QGIS 3.8
      */
-    static bool resampleSingleBandRaster( GDALDatasetH hSrcDS, GDALDatasetH hDstDS, GDALResampleAlg resampleAlg );
+    static bool resampleSingleBandRaster( GDALDatasetH hSrcDS, GDALDatasetH hDstDS, GDALResampleAlg resampleAlg, const char *pszCoordinateOperation = nullptr );
```

Comment:
I would make the operation non optional. I think there's other places calling this method which are also incorrectly ignoring the transform context...

Reply:
The only other place was in the postgresql provider, and it's only for doing a non reprojected resampling.

Nevertheless, I think you are right :)